tests/test_timeseriesflattener/test_feature_spec_objects.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import pandas as pd
 import pytest
+from psycop_feature_generation.loaders.raw.load_medications import clozapine, olanzapine
```

Comment:
Great that you're writing a test! 

However, we can't use the actual, private data for testing. The tests have to run on synthetic data, that we can have on our personal computers and GitHub for automated testing when there's a new PR. We should therefore not use anything from the `psycop-feature-generation-repo`. 

Instead, we have a `testing` folder that stores some loaders for synthetic data (`src/timeseriesflattener/testing/load_synth_data.py`). These also have the `n_rows` parameter, so it would be easy to write your tests to use them instead :)

Reply:
ah, of course not! thanks, I'll fix that 👍